### PR TITLE
relocate 40percent.club boards to new parent folder

### PR DESCRIPTION
**Pull request by zvecr**
As per the discussion on #4367, all the currently supported 40percent.club boards have be relocated to a new parent folder.

I have also had a quick stab at aligning the various readme files. I guess any aligning of code between boards should be done as a separate change to minimise change?

### CODE REVIEW

**Comment by mechmerlin** [comment]
I would have preferred this as separate PRs for each board you moved. That way if one board has build issues, it's not blocking the rest of them.

**Comment by zvecr** [comment]
Not a problem, i'm happy to submit each board separately as long as @drashna is OK with that vs the original proposal?

It seems the community layouts are failing for one of the boards, so at least it gives me time to fix the problem.

File: keyboards/40percentclub/4x4/rules.mk [hunk 1]
```diff
@@ -71,4 +70,5 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
 
-LAYOUTS = ortho_4x4  ortho_4x8  ortho_4x12  ortho_4x16 
+#FIXME: Community keymap build are currently failing due to missing functionality
+#LAYOUTS = ortho_4x4  ortho_4x8  ortho_4x12  ortho_4x16
```
Comment (drashna) [inline keyboards/40percentclub/4x4/rules.mk]: This is fine, actually.

File: keyboards/40percentclub/5x5/rules.mk [hunk 2]
```diff
@@ -71,4 +70,5 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
 
-LAYOUTS = ortho_5x5  ortho_5x10  ortho_5x15
+#FIXME: Community keymap build are currently failing due to missing functionality
+#LAYOUTS = ortho_5x5  ortho_5x10  ortho_5x15
```
Comment (drashna) [inline keyboards/40percentclub/5x5/rules.mk]: This is fine, actually.

**Comment by noroadsleft** [comment]
Re: community layouts failing to compile

## make 40percentclub/4x4:bakingpy

Fails to compile because the keymap file is configured to use the abbreviated TMK-style syntax, where the layout macro prepends `KC_` to every argument it receives. (@nooges y u do dis 2 me?) Adding a layout macro to support this behavior works. [Here's an example.](https://github.com/qmk/qmk_firmware/blob/a30dd2bb1711f5402007adf89af9b3adb758a7fb/keyboards/contra/contra.h#L30-L41)

If you fix that, you'll get an error:

```
error: 'BACKLIGHT_LEVELS' undeclared
```

Now you have to disable backlight via a conditional.


## make 40percentclub/4x4:guidoism

Fails because that keymap is configured for a Planck "MIT" layout, with a 2u space in the middle of the bottom row. Not sure how to fix this for a 4x4 in 4x12 configuration, because that's hardware side. The keyboard would have to be built with a 2u Point of Sale keycap to use that layout here.


## make 40percentclub/wanleg

Fails because it's being built with [`SWAP_HANDS_ENABLE = yes`](https://github.com/qmk/qmk_firmware/blob/a30dd2bb1711f5402007adf89af9b3adb758a7fb/layouts/community/ortho_4x12/wanleg/rules.mk#L2). His `rules.mk` file actually [turns the feature off for this keyboard](https://github.com/qmk/qmk_firmware/blob/a30dd2bb1711f5402007adf89af9b3adb758a7fb/layouts/community/ortho_4x12/wanleg/rules.mk#L8-L10), but moving the files means needing to update Line 8, because `$(strip $(KEYBOARD))` is now `40percentclub/4x4`.


## make 40percentclub:xyverz

Fails because it's being built with [`RGBLIGHT_ENABLE = yes`](https://github.com/qmk/qmk_firmware/blob/a30dd2bb1711f5402007adf89af9b3adb758a7fb/layouts/community/ortho_4x12/xyverz/rules.mk#L4-L8), but the 4x4 doesn't support that feature. A conditional is needed to turn it off here.


## Reference Changes

Feel free to steal any of the commits I added.

https://github.com/qmk/qmk_firmware/compare/fc477a1...noroadsleft:pull_4380

**Comment by nooges** [comment]
`nano` and `mf68` are also 40percent.club boards if you want to move them over.

**Comment by zvecr** [comment]
@nooges good catch on those additional boards, searching the repo with some of the readme contents also found the ut47.

@noroadsleft thanks for the diagnosis and most importantly, the fixes. I will roll them into this branch shortly.

**Comment by drashna** [comment]
@zvecr actually, if you would, leave the other keymaps alone.  It's fine if travis CI fails on community layouts.  As long as keyboard specific ones compile, especially the default keymaps, that's what's really important.

**Comment by zvecr** [comment]
@drashna no worries, i think i would prefer to leave that functionality disabled at the moment. Given the effort to effectively stub out the functionality i do not think this should cause an issue.

I should be receiving some pcbs shortly so would feel more comfortable adding all the macros to support the layouts option (ortho_4x4  ortho_4x8  ortho_4x12  ortho_4x16 ortho_5x5  ortho_5x10  ortho_5x15) when i can actually test and work out what to do with the mit planck layout.

**Comment by zvecr** [comment]
Right, i think I've done all i can currently, and opened up issue #4391 for something ive noted while moving the additional boards.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**